src/html/Quoting.h

```diff
@@ -6,16 +6,16 @@
  * Please see the COPYING and CONTRIBUTORS files for details.
  */
 
-#ifndef _SQUID_HTML_QUOTE_H
-#define _SQUID_HTML_QUOTE_H
+#ifndef SQUID__SRC_HTML_QUOTING_H
```

Comment:
why the double underscore in the header guard?

Reply:
> FYI, the use of two underscores helps reduce the complexity of differentiating these macros from the many other `SQUID_` macros around Squid.

If that is the goal, then we should find a better way to do that IMO.